lua/smh/client.lua

```diff
@@ -115,6 +115,7 @@ local function Setup()
     menuStreams.Output.Save:subscribe(function() SMH.Data.ShowSave = true end);
 
     menuStreams.Settings.Output.FreezeAll:subscribe(function(value) SMH.Data.FreezeAll = value end);
+	menuStreams.Settings.Output.TweenDisable:subscribe(function(value) SMH.Data.TweenDisable = value end);
```

Comment:
Convert tabs to spaces
```suggestion
    menuStreams.Settings.Output.TweenDisable:subscribe(function(value) SMH.Data.TweenDisable = value end);
```